### ISSUE TITLE
anybody succeded to install this whl file?  

### ISSUE THREAD

**Issue by Jiakui** [ZMpursue/rocm_pytorch_whl#1]
anybody succeded to install this whl file?

**Comment by ziliwang** [comment]
No, In my case, https://github.com/aieater/rocm_pytorch_informations#GPU-visibly-masking-and-multiple-GPUs work for me.

**Comment by Jiakui** [comment]
@ziliwang   thanks so much ,  I  will take a  look
